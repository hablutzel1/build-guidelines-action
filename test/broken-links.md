---
title: Broken Link Test
subtitle: Version X
author:

  - CA/Browser Forum

date: 7 January, 2021
copyright: |
  Copyright 2021 CA/Browser Forum

  This work is licensed under the Creative Commons Attribution 4.0 International license.
---

# 1. SECTION ONE

# 2. SECTION TWO

# 2.1 Test

This is a test of a [broken link](#section-four)

# 3. SECTION THREE

This is a test of a [good link](#3-section-three)

# 4. SECTION FOUR

This is a test of an [external link](https://tools.ietf.org/html/rfc5280#section-4.1)
that has a fragment identifier, but to another resource.
