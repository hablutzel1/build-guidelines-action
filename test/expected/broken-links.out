::group::Extract version
File broken-links.md is at version vX and commit 09f9e32
::endgroup::
::group::Linting Markdown
mdl -r MD032 /data/test/broken-links.md
::endgroup::
::group::Checking links
pandoc -f markdown+gfm_auto_identifiers --table-of-contents -s --no-highlight --lua-filter=/cabforum/filters/pandoc-list-table.lua --filter=/usr/bin/pantable -t gfm --lua-filter=/cabforum/filters/broken-links.lua -o /dev/null /data/test/broken-links.md
::error::Unable to resolve link to section-four
Valid identifiers are:

1-section-one
2-section-two
21-test
3-section-three
4-section-four
::endgroup::
